src/mutmap/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+find_package(Rscript)
+
+set(MUTMAP_DIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBEXECDIR}/mutmap)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/mutmap.sh.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/mutmap.sh" @ONLY)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/dist/build.R"
+  "${CMAKE_CURRENT_BINARY_DIR}/build.R" @ONLY)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/dist/bundle.js"
+  "${CMAKE_CURRENT_BINARY_DIR}/bundle.js" @ONLY)
+
```

Comment:
To see where I'm coming from, look at how I design web apps:

https://github.com/CartwrightLab/website/tree/master/apps

(You should have access to that directory.)